library/ceph_disks.py

```diff
@@ -1,176 +0,0 @@
-#!/usr/bin/python
```

Comment:
There was another version that Siva created which detected HDD as well. I think Vinit was of opinion that we should keep the library around in case ops guys need it or for future automation